testing/regress/hpcc/regression/cleanupReadme.md

```diff
@@ -0,0 +1,59 @@
+# Cleanup Parameter of Regression Suite run and query sub-command
```

Comment:
It would be better if this file stored in testing/regress directory.

Reply:
Yes, we shall move the file to the specified directory.